src/main/groovy/io/seqera/wave/controller/RegistryProxyController.groovy

```diff
@@ -272,9 +237,6 @@ class RegistryProxyController {
     }
 
     protected DigestStore manifestForPath(RoutePath route, HttpRequest httpRequest) {
-        // increase the quest counters
```

Comment:
can you remind me why these increases are not happening here any more?  thank you

Reply:
(also note that you can then get rid of the two lines in this file that contain references to `MeterRegistry`)

Reply:
We were using Prometheus to store the metrics, but now we are using DB to get them, so there is no need to have these functions.  
OK will remove the comment

Reply:
Makes sense, thanks for clarifying.

Note that `MeterRegistry` is also used in  `src/main/groovy/io/seqera/wave/service/builder/ContainerBuildServiceImpl.groovy`  for storing build metrics.  Is this also not required any more? -- I don't have full appreciation of this aspects yet. Feel free to double check with Paolo if you are in doubt.

Reply:
@pditommaso should we keep this build count based on MeterRegistry?
https://github.com/seqeralabs/wave/blob/03d18e5a6f234d21e3785e7416cff2dd459dac43/src/main/groovy/io/seqera/wave/service/builder/ContainerBuildServiceImpl.groovy#L237

Reply:
Think can be removed `meterRegistry` was never really used in practice

Reply:
Done